pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -541,7 +628,8 @@
     "runtime.win7-x64.runtime.native.System.IO.Compression": {
       "StableVersions": [
         "4.0.1"
-      ]
+      ],
+      "BaselineVersion": "4.3.0"
     },
     "runtime.win7-x86.runtime.native.System.Data.SqlClient.sni": {
```

Comment:
@gkhanna79 I dont understand what you mean by 
> Who did you work with on CoreFX team for those changes?  
We have checked in code to Native SNI as we have in the past. The assumption is that during a release the branch will be snapped. 

The tests dont fail because SNI is the networking layer for SqlClient and there are no tests in CI which talk to a server and hence the networking doesn't get exercised.